com.unity.render-pipelines.universal/Shaders/ComplexLit.shader

```diff
@@ -257,6 +259,8 @@ Shader "Universal Render Pipeline/Complex Lit"
             // -------------------------------------
             // Material Keywords
             #pragma shader_feature_local _NORMALMAP
+            #pragma shader_feature_local _PARALLAXMAP
+            #pragma shader_feature_local _ _DETAIL_MULX2 _DETAIL_SCALED
```

Comment:
Can any of these features be `_fragment` cc @lukaschod

Reply:
Sadly not, it requires viewDirectionTS. Unless we drop it and calculate in fragment, I do support that code path too

Reply:
#if (defined(_NORMALMAP) || (defined(_PARALLAXMAP) && !defined(REQUIRES_TANGENT_SPACE_VIEW_DIR_INTERPOLATOR))) || defined(_DETAIL)
#define REQUIRES_WORLD_SPACE_TANGENT_INTERPOLATOR
#endif